config/plugins/visualizations/h5web/package.json

```
@@ -0,0 +1,35 @@
+{
+    "name": "visualization",
+    "version": "0.1.0",
+    "keywords": [
+        "galaxy",
+        "visualization"
+    ],
+    "license": "AFL-3.0",
+    "dependencies": {
+        "@galaxyproject/charts": "^0.0.6",
+        "@h5web/app": "^6.6.1",
+        "@h5web/lib": "^6.6.1",
+        "@react-three/fiber": "7.0.26",
+        "axios": "1.2.0",
+        "axios-hooks": "4.0.0",
+        "backbone": "^1.3.3",
+        "normalize.css": "8.0.1",
+        "parcel-bundler": "^1.4.1",
+        "react": "17.0.2",
+        "react-dom": "17.0.2",
+        "react-dropzone": "14.2.3",
+        "react-icons": "4.7.1",
+        "react-router-dom": "6.3.0",
+        "three": "0.141.0"
```

FYI, I think a bunch of these dependencies can be removed:

```suggestion
        "backbone": "^1.3.3",
        "parcel-bundler": "^1.4.1",
        "react": "17.0.2",
        "react-dom": "17.0.2"
```